deepchem/feat/basic.py

```diff
@@ -85,6 +96,11 @@ def _featurize(self, mol):
     ----------
     mol : RDKit Mol
         Molecule.
+
+    Returns
+    -------
+    rval: np.ndarray
```

Comment:
This type is not correct.

Reply:
Maybe `return rval` -> `return np.asarray(rval)` is better

Reply:
Agree! Will swap to return array